### ISSUE TITLE
What is alternativee to --up flag 

### ISSUE THREAD

**Issue by Bosseur31** [OpenVPN/openvpn3-linux#74]
Hello,

I search an alternative for run a script at the start up VPN connection. In openvpn2 it was necessary to edit the .ovpn file with `up "/path/file/script/"`. But with openvpn3 this solution is outdated, an error appears `openvpn3-autoload[804]: ** WARNING ** Ignoring option: --up `.

Tanks for you help

**Comment by dsommers** [comment]
Yes, running scripts is not supported in OpenVPN 3; this is a security feature.

There are some possibilities to achieve running code at certain events with OpenVPN 3, but it depends on what you want to run.

Most users have used `--up` to do the DNS configuration.  This is however no longer needed in OpenVPN 3 Linux, as configuring DNS resolver settings is built-in.

**Comment by Bosseur31** [comment]
Thanks you for your response, i would like redirect VPN traffic for one user and use a up arguments for that. There is an alternative ?

**Comment by dsommers** [comment]
Alright, yes it is.  There is some NetworkChange D-Bus signals issues by the `openvpn3-service-netcfg` which can be used for this.  This is similar to how the AWS VPC integration is implemented.

There is a simple example Python test script you can look at to get an idea.
